services/authorization/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.2.1 (2024-05-22)
```

Comment:
it should be minor version, as we are introducing new functionality

Reply:
Changed version to 0.3.0